packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActions/PublishAction.tsx

```diff
@@ -241,26 +237,55 @@ interface TableRow {
 }
 
 const SelectedEntriesModalContent = ({
-  toggleModal,
-  refetchModalData,
-  setEntriesToFetch,
   setSelectedListViewEntries,
```

Comment:
why this needs to be passed from Publish action? you can get it from [here](https://github.com/strapi/strapi/pull/20234/files#diff-fc5c3b6fd53b68ac6c9556235411783d8d6e6c25cd807ce2a6d8ad474428dc8fR245)

Reply:
Needs to be passed because we have two different Table Context.

1. (Parent Table) List View Table
2. (Child Table) Modal Table

when we use `useTableContext` hook in the modal, we are accessing the context of the child table, not the parent, so we pass this function to update the selectedEntries in the parent table